content/en/docs/concepts/configuration/secret.md

```diff
@@ -337,16 +326,30 @@ A container using a Secret as a
 Secret updates.
 {{< /note >}}
 
-### Using Secrets as environment variables
+##### Automatic mounting of Secrets
+
+{{< feature-state for_k8s_version="v1.6" state="alpha" >}}
+
+You can arrange for Secrets to be automatically mounted into Pods. You can create
+a link between a Secret and a Pod using a PodPreset. The Secret, Pod, and
+PodPreset must all be part of the same {{< glossary_tooltip text="Namespace" term_id="namespace" >}}.
+See [Injecting Information into Pods Using a PodPreset](/docs/tasks/inject-data-application/podpreset/)
+for a detailed explanation of that process.
+
+
+#### Using Secrets as environment variables
 
 To use a secret in an {{< glossary_tooltip text="environment variable" term_id="container-env-variables" >}}
 in a Pod:
 
-1. Create a secret or use an existing one.  Multiple Pods can reference the same secret.
-1. Modify your Pod definition in each container that you wish to consume the value of a secret key to add an environment variable for each secret key you wish to consume. The environment variable that consumes the secret key should populate the secret's name and key in `env[].valueFrom.secretKeyRef`.
-1. Modify your image and/or command line so that the program looks for values in the specified environment variables.
+1. Create a Secret (or use an existing one).  Multiple Pods can reference the same Secret.
+1. Modify your Pod definition in each container that you wish to consume the value of a secret
+   key to add an environment variable for each secret key you wish to consume. The environment
+   variable that consumes the secret key should populate the secret's name and key in `env[].valueFrom.secretKeyRef`.
+1. Modify your image and/or command line so that the program looks for values in the specified
+   environment variables.
 
-This is an example of a Pod that uses secrets from environment variables:
+This is an example of a Pod that uses a Secret via environment variables:
```

Comment:
I think previous form was better as we're injecting 2 secrets in the example.

Reply:
The example injects two environment variables that both come from a single Secret (“mysecret”), so I think the switch to singular is justified.

Reply:
Misread GH diff :+1: